upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1468,58 +1042,6 @@ else
   echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
-# CRUS is being removed as part of this upgrade
-state_name="UNINSTALL_CRUS"
-state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
-if [[ $state_recorded == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
-  echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
-  {
-    # If CRUS is installed, uninstall it
-    if helm status -n services cray-crus; then
-      helm uninstall -n services cray-crus
-    fi
-  } >> "${LOG_FILE}" 2>&1
-  record_state "${state_name}" "$(hostname)" | tee -a "${LOG_FILE}"
-else
-  echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
-fi
-
-state_name="FINISH_NLS_UPGRADE"
-state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
-if [[ ${state_recorded} == "0" ]]; then
-  echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
-  {
-    # Restore labels and annotations on Argo CRDs
-    for c in $(kubectl get crd | grep argo | cut -d' ' -f1); do
-      kubectl label --overwrite crd "$c" app.kubernetes.io/managed-by="Helm"
-      kubectl annotate --overwrite crd "$c" meta.helm.sh/release-name="cray-nls"
-      kubectl annotate --overwrite crd "$c" meta.helm.sh/release-namespace="argo"
-    done
-
-    # CASMINST-6040 - need to wait for hooks.cray-nls.hpe.com crd to be created
-    # before uploading rebuild workflow templates which use hooks
-    echo "Wait for hooks.cray-nls.hpe.com crd to be created"
-
-    counter=0
-    until kubectl get crd hooks.cray-nls.hpe.com; do
-      counter=$((counter + 1))
-      sleep 5
-      # wait up to 90 seconds
-      if [[ ${counter} -gt 18 ]]; then
-        echo "ERROR: failed to find hooks.cray-nls.hpe.com crd"
-        exit 1
-      fi
-    done
-    set -e
-    "${locOfScript}/../../../workflows/scripts/upload-rebuild-templates.sh"
-    rpm --force -Uvh "$(find "${CSM_ARTI_DIR}"/rpm/cray/csm/ -name \*iuf-cli\*.rpm | sort -V | tail -1)"
-
-  } >> "${LOG_FILE}" 2>&1
-  record_state "${state_name}" "$(hostname)" | tee -a "${LOG_FILE}"
-else
-  echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
-fi
-
 state_name="PREPARE_KUBEADM"
```

Comment:
I think this is specific to k8s upgrade from `1.22` to `1.24`, so it can also be removed, at least for now - until k8s upgrade sequence is not worked out. 

@studenym-hpe @davidfluck-hpe - this stage was required, because in k8s `1.22` configmaps were named as `kubelet-config` and `kubeadm-config`. In k8s `12.4`, `kubelet-config` remains, but `kubeadm-config` is renamed to `kubeadm-config-1.24` (i.e. it becomes version specific).

Reply:
I will remove this for now.